config/default/foia_wizard.settings.yml

```
@@ -60,16 +60,16 @@ messages:
     value: "<p>Message 18</p>\r\n"
     format: rich_text
   m19:
-    value: "<p>Message 19</p>\r\n"
+    value: "<ul>\r\n\t<li>Contact your local field office. Please use the field office locator to find the office that services your zip code.</li>\r\n\t<li>If you are seeking your earnings records, please follow the instructions to complete Form SSA-7050-F4 available at: <a href=\"https://www.ssa.gov/forms/ssa-7050.pdf\">https://www.ssa.gov/forms/ssa-7050.pdf</a> and mail it to the address on the form along with the applicable fee and other evidentiary documents, as required.</li>\r\n</ul>\r\n\r\n<p>The following agency may also have the records you seek: SSA</p>\r\n\r\n<p>Can we help you find anything else?</p>\r\n"
```

Removing last paragraph as it will be in the Summary page functionality.
```suggestion
    value: "<ul>\r\n\t<li>Contact your local field office. Please use the field office locator to find the office that services your zip code.</li>\r\n\t<li>If you are seeking your earnings records, please follow the instructions to complete Form SSA-7050-F4 available at: <a href=\"https://www.ssa.gov/forms/ssa-7050.pdf\">https://www.ssa.gov/forms/ssa-7050.pdf</a> and mail it to the address on the form along with the applicable fee and other evidentiary documents, as required.</li>\r\n</ul>\r\n\r\n<p>The following agency may also have the records you seek: SSA</p>\r\n"
```